apps/dotcom/sync-worker/src/TLDrawDurableObject.ts

```diff
@@ -586,6 +604,7 @@ export class TLDrawDurableObject extends DurableObject {
 				if (this._isRestoring) return
 
 				const snapshot = JSON.stringify(room.getCurrentSnapshot())
+				this.maybeAssociateFileAssets()
```

Comment:
should probably also invoke this during `getRoom` if the room didn't exist yet and was successfully instantiated.

Reply:
Did it [here](https://github.com/tldraw/tldraw/commit/1e21ba9a153bbf18aefc78f35d4f5f3f8fabf2b6), let me know if that works.

Reply:
looks like it'll fail because the method is not bound?